src/java_bytecode/expr2java.cpp

```diff
@@ -83,7 +50,7 @@ std::string expr2javat::convert_code_function_call(
     unsigned p;
     std::string lhs_str=convert(src.lhs(), p);
 
-    // TODO: ggf. Klammern je nach p
+    // TODO: if necessery add parentheses, dependent on p
```

Comment:
Besides the typo (necessAry), looking at it more closely, I am also unsure about this. I tend to remove it.